README.md

```diff
@@ -1,10 +1,78 @@
+<!-- TODO(kramaranya): update when release [![Python Supported Versions](https://img.shields.io/pypi/pyversions/kubeflow.svg?color=%2334D058)](https://pypi.org/project/kubeflow/) -->
 
 ## Overview
 
-Kubeflow SDK streamlines user experience for data scientists and ML engineers to interact with
-various Kubeflow projects.
+Kubeflow SDK is a unified Python SDK that streamlines the user experience for AI Practitioners to interact with various
+Kubeflow projects. It provides simple, consistent APIs across the Kubeflow ecosystem, enabling users to focus on building
+ML applications rather than managing complex infrastrutcure.
 
-Design doc: https://docs.google.com/document/d/1rX7ELAHRb_lvh0Y7BK1HBYAbA0zi9enB0F_358ZC58w/edit
+> **Note**: The Kubeflow SDK is in active development. Currently, the SDK provides full support for **Kubeflow
+Trainer** with other components planned for future releases.
+
+### Kubeflow SDK Benefits
+
+- **Unified Experience**: Single SDK to interact with multiple Kubeflow components through consistent Python APIs
+- **Simplified ML Workflows**: Abstract away Kubernetes complexity, allowing ML practitioners to work in familiar Python environments
+- **Seamless Integration**: Designed to work together with all Kubeflow projects for end-to-end ML pipelines
+- **Local Development**: First-class support for local development requiring only `pip` installation
+
+## Installation
+
+Install the Kubeflow SDK using pip:
+
+```bash
+pip install git+https://github.com/kubeflow/sdk.git@main
+```
+
+## Supported Components
+
+| Component                   | Status | Description                                                    |
+|-----------------------------|--------|----------------------------------------------------------------|
+| **Kubeflow Trainer**        | ✅ **Available** | Train and fine-tune models with various frameworks   |
+| **Kubeflow Katib**          | 🚧 Planned | Hyperparameter optimization                                |
+| **Kubeflow Pipelines**      | 🚧 Planned | Build, run, and track ML workflows                         |
+| **Kubeflow Spark Operator** | 🚧 Planned | Run Apache Spark jobs on Kubernetes                        |
+| **Kubeflow Model Registry** | 🚧 Planned | Manage model artifacts, versions and ML artifacts metadata |
+| **Kubeflow Feast**          | 🚧 Planned | Feature store for ML feature management and serving        |
+| **Kubeflow KServe**         | 🚧 Planned | Model deployment and serving infrastructure                |
+
+## Community
+
+### Getting Involved
+
+- **Slack**: Join our [#kubeflow-trainer](https://www.kubeflow.org/docs/about/community/#kubeflow-slack-channels) Slack channel
+  <!-- TODO(kramaranya): update to #ml-experience once we kick off -->
+- **Meetings**: Attend the [AutoML and Training Working Group](https://bit.ly/2PWVCkV) bi-weekly meetings
+  <!-- TODO(kramaranya): update with kubeflow sdk meetings once we kick off -->
```

Comment:
We can change it to Kubeflow SDK call: https://docs.google.com/document/d/1jH2WAX2ePxOfI4JuiVK9nPlesDMiyg67xzLwhpR7wTQ/edit?tab=t.0